src/styles/getModalStyles.js

```diff
@@ -131,6 +133,32 @@ export default (type, windowDimensions, popoverAnchorPosition = {}) => {
             animationIn = 'fadeIn';
             animationOut = 'fadeOut';
             break;
+        case CONST.MODAL.MODAL_TYPE.CONTEXT_MENU:
+            modalStyle = {
+                ...modalStyle,
+                ...popoverAnchorPosition,
+                ...{
+                    position: 'absolute',
+                    alignItems: 'center',
+                    justifyContent: 'flex-end',
+                },
```

Comment:
I'm interested to know why we've been doing this in this file a lot, since I don't think it's necessary - I asked in [slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1623398725207000)